com.unity.netcode.gameobjects/Runtime/Interest/InterestNodeStatic.cs

```diff
@@ -0,0 +1,52 @@
+using System.Collections.Generic;
+using Unity.Netcode;
+using UnityEngine;
+
+namespace Unity.Netcode.Interest
+{
+    [CreateAssetMenu(fileName = "StaticInterestNode", menuName = "Interest/Nodes/Static", order = 1)]
```

Comment:
Since this menu item is applied to the Assets menu, `Interest/Nodes/Static` seems awkward from a user experience perspective. I would expect something like `Netcode/Interest/Static Interest Node`